backend/apps/matching/migrations/0002_permission_group_creation.py

```diff
@@ -0,0 +1,143 @@
+# Generated by Django 3.0.7 on 2020-06-15 12:28
+
+from django.db import migrations
+
+
+# This is added here and not in the project because these names are know to be valid only
+# in at this very moment in time - later migrations could alter them.
+# Thus we only need them here for now. Saving these as constants elsewhere for easy usability
+# in the project is a different thing, but probably unnecessary when we can use decorators.
+class NewPermissions:
+    can_approve_type_a = "can_approve_type_a"
```

Comment:
Are we really committed to hardcode everything as A and B everywhere in the code. Is that really the most elegant way?

Reply:
Like the previous comment, this is something we should have another design decisions call again soon, I think for now it is the most appropriate solution until someone drafts something better, so I stuck with it.